framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -346,6 +358,44 @@ else if(timeoutFailures.contains(test)){
         for (DrillTest test : executionFailures) {
           LOG.info("Query: " + test.getInputFile() + "\n" + test.getQuery());
           LOG.info("\nException:\n", test.getException());
+	  String [] localMsgArray = test.getException().getLocalizedMessage().split(":");
```

Comment:
Can you move this into a Util function? Basically it should pass the string and a regex and should return true or false.